src/qt/networkstyle.cpp

```diff
@@ -83,7 +83,8 @@ const NetworkStyle* NetworkStyle::instantiate(const std::string& networkId)
     std::string titleAddText = networkId == "liquidv1" ? "" : strprintf("[%s]", networkId);
     for (unsigned x=0; x<network_styles_count; ++x)
     {
-        if (networkId == network_styles[x].networkId)
+        // If it doesn't match any, use regtest since it's a custom chain
+        if (networkId == network_styles[x].networkId || x == network_styles_count - 1)
```

Comment:
This assumes at least one network style to not be an infinite loop. I guess that's ok :)